.github/workflows/ci.yaml

```diff
@@ -0,0 +1,45 @@
+name: Debos Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build Debos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go compiler
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.11
+        id: go
+
+      - name: Install required packages
+        run: sudo apt-get update ; sudo apt-get install --yes libostree-dev
+
+      - name: Checkout debos source
+        uses: actions/checkout@v2
+
+      - name: Build debos on host
+        env:
+          GOSRC: /home/runner/go/src
+          GOBIN: /home/runner/go/bin
+        run: |
+          mkdir -p ${GOSRC}/github.com/go-debos
+          ln -s $(pwd) ${GOSRC}/github.com/go-debos/debos
+          go get -v ./...
```

Comment:
A lot of this duplicates what `docker/Dockerfile` already does; Which i guess is fine but it will cause duplication going forward ; Could we do this in a smarter way ?

Reply:
removed the duplication !